static/css/ghe-loading.css

```diff
@@ -0,0 +1,45 @@
+.gheLoading___body {
+    background: #fff;
+}
+
+.gheLoading___container {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+    margin: auto;
+    max-width: 648px;
+}
+
+.gheLoading___header {
+    font-size: 1.5rem;
+}
+
+.gheLoading___imgDivider {
+    font-size: 2.8rem;
+    color: #6B778C;
+}
+
+.gheLoading___imgContainer {
+    margin: 4.688em 0 4.063em;
```

Comment:
Do we need to be this precise?

Reply:
Not necessarily. I simply converted the pixels to em and added those values.
For avoiding any confusion, rounding off these values.

Reply:
we could probably just use pixels in that case then?

Reply:
Following the following convention for GitHub for Jira: 
- px for borders and max-width of a section
- em for padding and margins (so it is relative to the parent container)
- rem for font size (so it's relative to the root unit and stays consistent regardless of other measurements used elsewhere)